app/lib/presentation/screens/home.dart

```diff
@@ -71,57 +71,82 @@ class Home extends StatelessWidget {
                     itemBuilder: (context, index) {
                       return Padding(
                         padding: const EdgeInsets.all(8.0),
-                        child: Card(
-                          child: ListTile(
-                            title: Text(
-                              state.accounts[index].accountName,
-                              style:
-                                  const TextStyle(fontWeight: FontWeight.bold),
-                            ),
-                            subtitle: Text(
-                              '${state.accounts[index].accountNumber.toString()} - ${state.accounts[index].bankName}',
-                            ),
-                            trailing: Row(
-                              mainAxisSize: MainAxisSize.min,
-                              children: [
-                                GestureDetector(
-                                  onTap: () {
-                                    // copy data to clipboard
-                                    Clipboard.setData(
-                                      ClipboardData(
-                                        text: state
-                                            .accounts[index].accountNumber
-                                            .toString(),
-                                      ),
-                                    );
-                                    ScaffoldMessenger.of(context).showSnackBar(
-                                      const SnackBar(
-                                        content: Text("Copied to clipboard"),
-                                      ),
-                                    );
-                                  },
-                                  child: const Icon(
-                                    Icons.copy,
-                                    color: Colors.deepPurple,
-                                  ),
+                        child: Dismissible(
+                          confirmDismiss: (direction) => showDialog<bool>(
```

Comment:
Is there a way to specify and limit which direction the action happens in? Like swiping in one direction should be deletion and we can reserve the other direction for something else.

Reply:
I made the changes to only allow swiping from left to right